functions.Makefile

```diff
@@ -300,13 +300,13 @@ $(1)-terraform-import:
 	$(ROOT)/docker_run.py --aws -- \
 		--volume $(ROOT):$(ROOT) \
 		--workdir $(ROOT)/$(2) \
-		hashicorp/terraform:0.11.7 import $(filter-out $(1)-terraform-import,$(MAKECMDGOALS))
+		hashicorp/terraform:0.11.10 import $(filter-out $(1)-terraform-import,$(MAKECMDGOALS))
 
 $(1)-terraform-state-rm:
 	$(ROOT)/docker_run.py --aws -- \
 		--volume $(ROOT):$(ROOT) \
 		--workdir $(ROOT)/$(2) \
-		hashicorp/terraform:0.11.7 state rm $(filter-out $(1)-terraform-state-rm,$(MAKECMDGOALS))
+		hashicorp/terraform:0.11.10 state rm $(filter-out $(1)-terraform-state-rm,$(MAKECMDGOALS))
```

Comment:
You need to update the `terraform_wrapper` image tags to match.

You can see the list of available tags here: https://hub.docker.com/r/wellcome/terraform_wrapper/tags/

The numbers come from the Travis build numbers, so they're monotonically increasing: https://travis-ci.org/wellcometrust/dockerfiles/builds/448308416